app/views/queue_items/index.html.haml

```diff
@@ -0,0 +1,33 @@
+%section.my_queue.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %h2 My Queue
+        %table.table
+          %thead
+            %tr
+              %th(width="10%") List Order
+              %th(width="30%") Video Title
+              %th(width="10%") Play
+              %th(width="20%") Rating
+              %th(width="15%") Genre
+              %th(width="15%") Remove
+          %tbody
+            - @queue_items.each do |queue_item|
+              %tr
+                %td= queue_item.position
+
+                %td
+                  = link_to("#{queue_item.video_title}", video_path(queue_item.video))
+                %td
+                  = button_to "Play", nil, class: "btn btn-default"
+                %td
+                  = queue_item.rating
+                %td
+                  = link_to("#{queue_item.category.name}", category_path(queue_item.category))
```

Comment:
You can use `queue_item.category` shortcut instead `category_path(queue_item.category)`.